src/traffic_server/InkAPI.cc

```diff
@@ -8428,594 +8431,159 @@ TSHttpTxnConfigStringGet(TSHttpTxn txnp, TSOverridableConfigKey conf, const char
   return TS_SUCCESS;
 }
 
-// This is pretty suboptimal, and should only be used outside the critical path.
+// This is a map of all overridable configurations, with the data type
+static const std::unordered_map<std::string_view, std::tuple<TSOverridableConfigKey, TSRecordDataType>> _overridable_map(
```

Comment:
This is an invalid use of a reserved name - "all identifiers that ... begin with an underscore are reserved for use as names in the global namespace". This means you can use a leading underscore in a namespace or class, but not at file or global scope.